src/main/java/games/strategy/triplea/oddscalc/DummyPlayer.java

```diff
@@ -0,0 +1,230 @@
+package games.strategy.triplea.oddscalc;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+import games.strategy.net.GUID;
+import games.strategy.triplea.ai.AIUtils;
+import games.strategy.triplea.ai.AbstractAI;
+import games.strategy.triplea.delegate.DiceRoll;
+import games.strategy.triplea.delegate.Matches;
+import games.strategy.triplea.delegate.MustFightBattle;
+import games.strategy.triplea.delegate.dataObjects.CasualtyDetails;
+import games.strategy.triplea.delegate.dataObjects.CasualtyList;
+import games.strategy.triplea.delegate.remote.IAbstractPlaceDelegate;
+import games.strategy.triplea.delegate.remote.IMoveDelegate;
+import games.strategy.triplea.delegate.remote.IPurchaseDelegate;
+import games.strategy.triplea.delegate.remote.ITechDelegate;
+import games.strategy.util.Match;
+
+/**
+ * Do-nothing implementation to support battle calc.
+ */
+public class DummyPlayer extends AbstractAI {
```

Comment:
This code is all just moved, it was from an inner class, same with any of the other Dummy* classes.